Content.Shared/Whitelist/EntityWhitelist.cs

```diff
@@ -44,8 +44,15 @@ public sealed partial class EntityWhitelist
     [DataField]
     public List<ProtoId<ItemSizePrototype>>? Sizes;
 
-    [NonSerialized, Access(typeof(EntityWhitelistSystem))]
-    public List<ComponentRegistration>? Registrations;
+    [NonSerialized]
+    private List<ComponentRegistration>? _registrations;
+
+    [Access(typeof(EntityWhitelistSystem))]
+    public List<ComponentRegistration>? Registrations
+    {
+        get => _registrations;
+        set => _registrations = value;
+    }
```

Comment:
why

Reply:
ok real that's what i get for not double checking the diff
leftover from me fucking up the storage containers